src/main/java/com/softserve/actent/constant/StringConstants.java

```diff
@@ -14,4 +14,8 @@
     public static final String CITY_NO_LONGER_THAN_THIRTY_SYMBOLS ="City shouldn`t be longer than 30";
     public static final String REGION_NO_LONGER_THAN_THIRTY_SYMBOLS ="Region shouldn`t be longer than 30";
     public static final String COUNTRY_NO_LONGER_THAN_THIRTY_SYMBOLS ="Country shouldn`t be longer than 30";
+
+    public static final String TITLE_SHOULD_NOT_BE_BLANK = "Title can`t be blank";
```

Comment:
Merge this before Arsen's (or who is that panda :) ) merge.

then tell Arsen to merge develop to his branch locally, to avoid merge conflicts (if he is going to change this java class to .property one)